opensearchpy/connection/http_urllib3.py

```diff
@@ -279,7 +279,11 @@ def perform_request(
             self.log_request_fail(
                 method, full_url, url, orig_body, duration, response.status, raw_data
             )
-            self._raise_error(response.status, raw_data)
+            self._raise_error(
+                response.status,
+                raw_data,
+                response.headers.get("Content-Type", None),
```

Comment:
create a new method to get this values as
```     
     def get_response_headers(self, response)
          response_headers = {
                header.lower(): value for header, value in response.headers.items()
            }
```
and later
```
  self._raise_error(
      response.status,
      raw_data,
      self.get_response_headers(.get("content-type", "")),
```

Reply:
Done